packages/shelter/src/index.ts

```diff
@@ -1,5 +1,5 @@
 import * as patcher from "spitroast";
-import * as ui from "@uwu/shelter-ui";
+import * as ui from "../../shelter-ui";
```

Comment:
```suggestion
import * as ui from "@uwu/shelter-ui";
```

shelter UI is a separate package now, leave it as such. If you are using pnpm it will link to the local version just fine without this.